packages/web/src/components/layouts/SearchResults/SearchResults.tsx

```diff
@@ -183,69 +241,45 @@ export default function SearchResults() {
             name='pinterest.com'
             totalPackages={6}
             lastScanDate='21 feb in 21:30'
+            loading={loading}
           />
 
           <aside className={styles.sidebar}>
             <div className={styles.sidebarItem}>
-              <div className={styles.meta}>
-                <div className={styles.metaItem}>
-                  <span className={styles.metaIcon}>
-                    <Icon kind='weight' width={24} height={24} />
-                  </span>
-                  <span className={styles.metaText}>159 kb webpack bundle size</span>
-                </div>
-                <div className={styles.metaItem}>
-                  <span className={styles.metaIcon}>
-                    <Icon kind='search' width={24} height={24} color='#212121' />
-                  </span>
-                  <span className={styles.metaText}>50 scripts found</span>
-                </div>
-                <div className={styles.metaItem}>
-                  <span className={styles.metaIcon}>
-                    <Icon kind='bug' width={24} height={24} color='#F3512E' />
-                  </span>
-                  <span className={styles.metaText}>6 vulnerabilities in 4&nbsp;packages</span>
-                </div>
-                <div className={styles.metaItem}>
-                  <span className={styles.metaIcon}>
-                    <Icon kind='duplicate' color='#F3812E' width={24} height={24} />
-                  </span>
-                  <span className={styles.metaText}>12 duplicate packages</span>
-                </div>
-                <div className={styles.metaItem}>
-                  <span className={styles.metaIcon}>
-                    <Icon kind='outdated' color='#F1CE61' stroke='white' width={24} height={24} />
-                  </span>
-                  <span className={styles.metaText}>18 outdated packages</span>
-                </div>
-              </div>
+              <SidebarMeta meta={metaItems} loading={loading} />
             </div>
 
             <div className={styles.sidebarItem}>
               <SidebarCategory
+                categoryName='Keywords'
                 keywordsList={keyWords}
                 selectedKeywords={selectedKeywords}
                 selectHandler={handleKeywordsChange}
                 renderComponent='chip'
```

Comment:
nit: What about splitting these off into separate components and passing them back here through `children` or a render prop of sorts?